CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
+### New Features
+ - Team interface support
+
+### Bug Fixes
+ - Fix memory leak of NM.Client
+ - Get SR-IOV VFs configuration from runtime
+ - Fix DHCPv4 with static IPv6
+ - Learn the OVS Bridge ports and interfaces from NetworkManager
+
```

Comment:
Missing is something like:
- Wait a short time before verification to allow the network state to stabilize

Reply:
Added. Thank you for point it out :)